core/kazoo_number_manager/src/carriers/knm_bandwidth2.erl

```diff
@@ -54,33 +55,48 @@
 
 -ifdef(TEST).
 -define(BW2_ACCOUNT_ID, "eunit_testing_account").
+-define(IS_SANDBOX_PROVISIONING_TRUE, 'true').
+-define(IS_PROVISIONING_ENABLED, 'true').
+-define(BW2_ORDER_NAME_PREFIX, "Kazoo").
+-define(BW2_SITE_ID, "").
+-define(BW2_SIP_PEER, "").
+-define(BW2_API_USERNAME, <<>>).
+-define(BW2_API_PASSWORD, <<>>).
+-define(MAX_SEARCH_QUANTITY, 500).
 -else.
--define(BW2_ACCOUNT_ID,
-        kapps_config:get_string(?KNM_BW2_CONFIG_CAT, <<"account_id">>, "")).
--endif.
 
--define(IS_SANDBOX_PROVISIONING_TRUE,
-        kapps_config:get_is_true(?KNM_BW2_CONFIG_CAT, <<"sandbox_provisioning">>, 'true')).
--define(IS_PROVISIONING_ENABLED,
-        kapps_config:get_is_true(?KNM_BW2_CONFIG_CAT, <<"enable_provisioning">>, 'true')).
--define(BW2_ORDER_NAME_PREFIX,
-        kapps_config:get_string(?KNM_BW2_CONFIG_CAT, <<"order_name_prefix">>, "Kazoo")).
-
--define(BW2_API_USERNAME,
-        kapps_config:get_binary(?KNM_BW2_CONFIG_CAT, <<"api_username">>, <<>>)).
--define(BW2_API_PASSWORD,
-        kapps_config:get_binary(?KNM_BW2_CONFIG_CAT, <<"api_password">>, <<>>)).
--define(BW2_SIP_PEER,
-        kapps_config:get_string(?KNM_BW2_CONFIG_CAT, <<"sip_peer">>, "")).
--define(BW2_SITE_ID,
-        kapps_config:get_string(?KNM_BW2_CONFIG_CAT, <<"site_id">>, "")).
-
--define(MAX_SEARCH_QUANTITY,
-        kapps_config:get_pos_integer(?KNM_BW2_CONFIG_CAT, <<"max_search_quantity">>, 500)).
```

Comment:
Please revert these kapps config changes.
This is not how testing works and this is not the only place it's done this way.